content/zh/docs/reference/kubernetes-api/cluster-resources/namespace-v1.md

```diff
@@ -0,0 +1,807 @@
+---
+api_metadata:
+  apiVersion: "v1"
+  import: "k8s.io/api/core/v1"
+  kind: "Namespace"
+content_type: "api_reference"
+description: "Namespace 为 NAMES 提供了一个作用域。"
+title: "Namespace"
+weight: 2
+auto_generated: true
+---
+
+`apiVersion: v1`
+
+`import "k8s.io/api/core/v1"`
+
+## Namespace {#Namespace}
+<!--
+Namespace provides a scope for Names. Use of multiple namespaces is optional.
+-->
+Namespace 为 Names 提供了作用域。使用多个命名空间是可选的。
+<hr>
+
+- **apiVersion**: v1
+
+- **kind**: Namespace
+
+- **metadata** (<a href="{{< ref "../common-definitions/object-meta#ObjectMeta" >}}">ObjectMeta</a>)
+<!--
+  Standard object's metadata. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
+-->
```

Comment:
Without indention spaces, the following paragraph won't be at the same level with list item.
```suggestion
  <!--
  Standard object's metadata. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
  -->
```

Reply:
Thank you for your comments. The translation has been improved according to the review